docs/УправлениеИнтеграциями.md

```
@@ -0,0 +1,276 @@
+
+**Пример записи истории интеграции:**
+
+```bsl
+Процедура Тест(Знач Сессия)
+
+    ИнтеграционныйПоток = Справочники.пбп_ИнтеграционныеПотоки.НайтиПоРеквизиту(
+        "ИдентификаторНастройки", "ПолучениеТоваровНаСкладахИзСистемыN");
+
+    РеквизитыПотока = пбп_ОбщегоНазначенияСервер.ЗначенияРеквизитовОбъекта(ИнтеграционныйПоток,
+        "ТочкаВхода, НастройкаИнтеграции, НастройкаИнтеграции.ИнтегрируемаяСистема");
+	
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)